pegasus/sites.v3/code.org/views/global/fa/resources_tabs_content_assessments.haml

```diff
@@ -8,5 +8,5 @@
         =hoc_s(:resources_tabs_assessments_subhead)
     %p.body-three
       =hoc_s(:resources_tabs_assessments_desc)
-    %a.link-button{href: CDO.studio_url("/s/csd1-2023/lessons/8"), target: "_blank", rel: "noopener noreferrer"}
+    %a.link-button{href: CDO.studio_url("/s/courseb-2022/lessons/13?lang=fa-IR"), target: "_blank", rel: "noopener noreferrer"}
```

Comment:
Is this the right way to enforce the language? I thought it was `/lang/fa-IR`. Maybe something for @wilkie to confirm!

Reply:
When I type that in it seems to switch it to https://studio.code.org/s/courseb-2022/lessons/1?lang=fa-IR

Reply:
cool sounds good!

Reply:
If I go to https://studio.code.org/s/courseb-2022/lessons/1?lang=fa-IR in an incognito window, I'm still in English though

Reply:
Hmm ya. Lets wait for wilkie to confirm